src/test/java/org/fundacionjala/coding/christian/ChristianKata14Test.java

```diff
@@ -0,0 +1,60 @@
+package org.fundacionjala.coding.christian;
+
+import org.junit.Before;
+import org.junit.Test;
+import java.util.Arrays;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Performed by Christian.
+ */
+public class ChristianKata14Test {
+    private static final int NEGATIVE_TWO = -2;
+    private static final int NEGATIVE_TEN = -10;
+    private static final double NEGATIVE_FOURP = -4.5;
+    private static final int NUMBER_2 = 2;
+    private static final int NUMBER_0 = 0;
+    private static final int NUMBER_4 = 4;
+    private static final int NUMBER_1 = 1;
+    private static final int NUMBER_3 = 3;
+    private static final int NUMBER_5 = 5;
+
+    private ChristianKata14 katita;
+    /**
+     *
+     */
+    @Before
+    public void setup() {
+        katita = new ChristianKata14();
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void basicTests() {
+        assertEquals(Arrays.toString(new double[]{NUMBER_2, NUMBER_2, NUMBER_2, NUMBER_2 }),
```

Comment:
Don't create static variable for the number
you can directly declare local final array variables.